### ISSUE TITLE
Description on type in schema.prisma not added to GraphQL

### ISSUE THREAD

**Issue by michaelpoellath** [unlight/prisma-nestjs-graphql#157]
First of, love the project, great work here, helps us a lot!

We have multiple models and types in our schema.prisma file with many comments. 
When generating GraphQL classes we are missing all doc strings on the types.

Sample schema.prisma:
```
## Descriptions lost
/// NameValueUnit explains itself
type NameValueUnit {
  ///  Name is foo
  name  String?
  ///  Value is bar
  value Float?
  /// Unit is baz
  unit  String?
}

## Works as expected, descriptions are on GraphQL Classes
/// testing
model Test {

  /// NameValueUnit
  nvu NameValueUnit
}
```

Am I doing something wrong?
Is it possible to add these or is this an upstream issue with Prisma?

**Comment by unlight** [comment]
Indeed. Generated Data Model Meta Format (does not contain comments for type blocks.

**Comment by unlight** [comment]
There is a 3rd party schema parser - https://github.com/MrLeebo/prisma-ast
But it doesnt see composite types.

**Comment by MrLeebo** [comment]
@unlight hey there 👋 

I'm the author of prisma-ast. I wanted to drop in and mention that prisma-ast now supports composite types as of v0.9.0. If you notice any other prisma objects that are missing, don't hesitate to leave me a note. I may not know about them otherwise.
